docs/source/gbd2017_models/risks/low_birthweight_short_gestation/index.rst

```diff
@@ -331,16 +335,19 @@ All-cause mortality is the sum of all the cause-specific mortality rates
     - \sum\limits_{\text{blue}}\text{CSMR}
 
   This equation can be substituted into :eq:`mortality_hazard` below to
-  compute the mortality hazard for an individual simulant.
+  eliminate the pink causes from the computation of the mortality hazard for an
+  individual simulant.
 
-To compute the individual mortality hazard :math:`\text{mr}(i)`, for the
-modelled causes (green and salmon) we will need the cause-state-dependent excess
-morality rates (EMR) instead of the CSMR, which is the average EMR over all
-cause states. For example, the excess mortality rates for a two-state cause
-(with condition / without condition) would be:
+We now describe our strategy for defining the individual mortality hazard
+:math:`\text{mr}(i)`, taking an individual's LBWSG category into account. For
+the modelled causes (green and salmon) we will use the cause-state-dependent
+excess morality rates (EMRs) instead of the CSMR, which is the average EMR over
+all cause states (including the "without condition" state). For example, the
+excess mortality rates for a two-state cause (with condition / without
+condition) would be:
```

Comment:
I'm wondering if we can clarify these sentences a bit: 
original:
For the modelled causes (green and salmon) we will use the cause-state-dependent
excess morality rates (EMRs) instead of the CSMR, which is the average EMR over
all cause states (including the "without condition" state). For example, the
excess mortality rates for a two-state cause (with condition / without
condition) would be.

suggested:
For the modelled causes (green and salmon) we will use the excess morality rates (EMRs) instead of the CSMR. The EMR is cause-state dependent while the CSMR is the average EMR over all cause-states. For example, the excess mortality rates for a two-state cause (with condition / without condition) would be:

Reply:
Sure, that sounds good. What about keeping the "without condition" clarification in, like this:

> For the modelled causes (green and salmon) we will use the excess morality rates (EMRs) instead of the CSMR. The EMR is cause-state dependent while the CSMR is the average EMR over all cause-states (including the "without condition" state). For example, the excess mortality rates for a two-state cause (with condition / without condition) would be:

I just thought it was a good idea to really highlight this point because (1) it might not be clear that "cause-state" includes the state where you _don't_ have the cause, and (2) I think it helps clarify the distinction between the EMRs and the CSMRs.

Reply:
oh yes, thank you for the clarification. I originally read 'with' condition state, sorry. It's very clear now :)